index.d.ts

```diff
@@ -0,0 +1,33 @@
+interface AbortError extends Error {}
+
+interface AbortErrorConstructor {
+	/**
+	 * Abort pending execution. All unresolved promised are rejected with a `AbortError` error.
+	 */
+    new(): AbortError;
+    readonly prototype: AbortError;
+}
+
+export const AbortError: AbortErrorConstructor;
```

Comment:
Why can't we just do `export class AbortError extends Error {}`?

Reply:
Good point! I'll check it out tomorrow. I looked at how Node.js did it and this is what they used but it looks like that should work as well.